dockers/docker-orchagent/kernel_arp_restore.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+function wait_for_intf_up {
+    INTF_NAME=$1
+
+    until [[ `ip link show $INTF_NAME | grep 'state UP'` ]]; do
+        sleep 1;
+    done
+}
+
+function restore_arp_to_kernel {
+    ARP_FILE='/arp.json.1'
+    NUM_ENTRIES=`jq 'length' $ARP_FILE`
+
+    for i in $( seq 0 $(($NUM_ENTRIES - 1)) ); do
+
+        # For the ith object, get the first key
+        # 'jq' sorts the keys by default so this should always be
+        # the 'NEIGH_TABLE' key, not the 'OP' key
+        KEY=`jq ".[$i] | keys[0]" $ARP_FILE`
```

Comment:
can we use [`startwith`](https://stedolan.github.io/jq/manual/#startswith(str)) here?

Reply:
Do you mean to confirm that the KEY we select is actually the `NEIGH_TABLE` key?

Reply:
Yes, relying on the key would come first sounds like asking for troubles.

Reply:
Fixed